common/nodejs-utils/src/distribution.rs

```diff
@@ -92,7 +92,7 @@ impl Distribution {
     fn active_requirement(self) -> anyhow::Result<Requirement> {
         Requirement::parse(match self {
```

Comment:
would it be safer to create these as a set of version range objects instead of relying on parsing?  or possibly making the parser smart enough to recognize that the input string was invalid and should have thrown an error here?

Reply:
Yeah, probably safer. Quite a bit more verbose and harder to read. I'd also have to open up the `Version` API quite a bit.